### PR TITLE
Added is prime

### PR DESCRIPTION
**Pull request by florpi**
Function to check whether number is prime. Added unit tests.

### CODE REVIEW

File: Cuesta/template/simple_functions.py [hunk 1]
```diff
@@ -10,3 +12,28 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
```
Comment (tahumada) [inline Cuesta/template/simple_functions.py:15]: I'll take a look :)

**Comment by stephenportillo** [comment]
I'll review this!

File: Cuesta/template/simple_functions.py (hunk 1 above)
Comment (tahumada) [reply Cuesta/template/simple_functions.py:15]: It works!

**Review by tahumada — APPROVED** [review]

**Review by stephenportillo — APPROVED** [review]
Looks good to me too!

**Comment by bsipocz** [comment]
@florpi @tahumada  @stephenportillo  - Thanks for opening the pull request and for the review(s)! Everything looks good, so we'll go ahead and merge this. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code.

@tahumada - your comments are generic and not about the specific code changes in the "Files changed" tab. For these cases it's possible to leave the generic comments when you set the review status (Approve/Require Changes/etc.), or in the conversation tab, here rather than inline.

**Pull request merged**